lib/image-stream/utils.js

```diff
@@ -37,6 +38,10 @@ const archiveType = require('archive-type');
  */
 exports.getArchiveMimeType = (file) => {
 
+  if (mime.lookup(file)) {
```

Comment:
Yeah, I guess – but probably should do that in a separate, preflight PR – what do you think?